libraries/chain/database.cpp

```diff
@@ -64,8 +64,8 @@ const uint8_t streaming_platform_object::type_id;
 
 muse::chain::asset_id_type MUSE_SYMBOL=(muse::chain::asset_id_type(0));
 muse::chain::asset_id_type VESTS_SYMBOL=(muse::chain::asset_id_type(1));
-muse::chain::asset_id_type MBD_SYMBOL=(muse::chain::asset_id_type(2));
-    
+muse::chain::asset_id_type MBD_SYMBOL=(muse::chain::asset_id_type(13)); //changing MBD id from 2 to 13
```

Comment:
Don't do that. This will retroactively change `MBD_SYMBOL`, which means that you cannot replay the existing blockchain. What you can do is add a new `XUSD_SYMBOL`, and change the code to use `XUSD_SYMBOL` or `MBD_SYMBOL` depending on whether it's before or after the consensus upgrade.

Also, you shouldn't use fixed IDs here. The 3 base assets are created by the core code automatically and therefore have known IDs, but anything after these will be assigned an ID upon creation. One way to deal with this could be to register the assets manually both in testnet and in mainnet, and then insert the resulting IDs into the codebase (which will likely be different between mainnet and testnet).